frontend/src/utils/util.ts

```diff
@@ -197,3 +197,11 @@ export function combineBoardInfo(boardInfoResponses: BoardInfoResponse[]) {
     };
   }
 }
+
+export const xAxisLabelDateFormatter = (dateRange: string) => {
+  const [startDate, endDate] = dateRange.split('-');
+  const startMonthDay = startDate.slice(5);
```

Comment:
do we need to check the length of dateRange, startDate, endDate?

Reply:
No need, the format of time must be similar to 2024/03-05/2024/03/15.The length of time is fixed.